root/url/Header.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+// Copyright (C) 2017 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('../components/EntityTabs');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+
+const URLHeader = ({url, page, entity_properties}) => {
+
+  const urlEntity = <EntityLink content={url.decoded} entity={url} subPath={'show'} />;
+
+  let urlObj = urlEntity;
+  if (url.edits_pending)
+    urlObj = (
+        <span className="mp">
+          {urlEntity}
+        </span>
+      );
+
+  let subHeading = l('URL');
+  if (url.type_name)
+    subHeading = url.type_name;
```

Comment:
I am not sure where this is defined. Can anyone help me with this?

Reply:
`type_name` is not defined for URL entities, it is defined for Artist, Event, Series, and Work.
There is no plan to specify `type` for URLs since their roles are actually specified with relationship types.
Just drop that!